### ISSUE TITLE
Массовая привязка-отвязка связей в админке

### ISSUE THREAD

**Issue by goldman-sax** [instantsoft/icms2#1397]
**Используемая версия InstantCMS**: 2.15.1

Хорошо бы иметь возможность в админке массово привязывать список новостей к конкретной статье, т.е. выделить нужные статьи списком и привязать. Чтобы такая опция появилась на скриншоте.
Это сэкономит массу времени при работе со связями.
![2022-01-31_124142](https://user-images.githubusercontent.com/58044217/151756082-486601b8-fb34-4f2f-82d4-ce4357af9055.jpg)

**Comment by vlad7090** [comment]
Да. Также нужно иметь возможность массово удалять категории в админке) главное. чтобы система уведомила о том, что там могут быть записи, которые предложила бы перенести в корень ТК.
